website/content/en/docs/cli/operator-sdk_generate_bundle.md

```diff
@@ -30,62 +38,55 @@ operator-sdk generate bundle [flags]
 
 ```
 
-  # Generate bundle files and build your bundle image with these 'make' recipes:
```

Comment:
IMO we need here: 
- Clarifies that if we are using an SDK project you can use the target instead `make bundle`
- If we are running outside of the project or for a project that does not follow up the SDK layout (Non-SDK) : example.

Reply:
I kinda want to keep the command docs about the command only, and link to the OLM integration docs for in-project usage.

Reply:
Why OLM integration docs?
IMO we need to clarify who/when can use the command and what are the circumstances.

I do not think that has a relation with OLM docs at all. OLM docs would link sdk docs instead for users to know how SDK CLI can help them to integrate their projects when it is SDK layout and when is not.

Reply:
>I do not think that has a relation with OLM docs at all.

Not OLM's docs, the SDK's [OLM integration docs](sdk.operatorframework.io/docs/olm-integration/)

Reply:
It is fine. However, I think we cannot close the #4442 yet with this PR. 
we need to:
- Address https://github.com/operator-framework/operator-sdk/issues/4528
- Then, add the link here.